package.json

```diff
@@ -3,27 +3,33 @@
   "version": "4.9.45",
   "description": "Dark mode for every website",
   "scripts": {
-    "api": "node tasks/build.js --api",
     "benchmark-server": "node tests/benchmark-server/index.js",
+    "api": "node tasks/build.js --api",
```

Comment:
Revert this? these should be alphabetic order.

Reply:
Sure, I can do that. Sorry, I didn't notice alphabetical order. I was grouping them by "build", "test", etc.